diff -up num_cpus-1.13.0/Cargo.toml.1~ num_cpus-1.13.0/Cargo.toml
--- num_cpus-1.13.0/Cargo.toml.1~	2021-03-25 04:04:30.077578531 +0100
+++ num_cpus-1.13.0/Cargo.toml	2021-03-25 04:05:10.640825887 +0100
@@ -23,5 +23,3 @@ license = "MIT/Apache-2.0"
 repository = "https://github.com/seanmonstar/num_cpus"
 [dependencies.libc]
 version = "0.2.26"
-[target."cfg(all(any(target_arch = \"x86_64\", target_arch = \"aarch64\"), target_os = \"hermit\"))".dependencies.hermit-abi]
-version = "0.1.3"
